tests/integration/ie-tests.sh

```diff
@@ -14,57 +14,69 @@ test_example_job() {
 
   echo $K8S_MASTER_URL
 
+  PREVIOUS_JOB=$(kubectl --kubeconfig=${KUBE_CONFIG} get pods | grep driver | tail -n 1 | cut -d' ' -f1)
+
   # run the sample pi job using spark-submit
-  spark-client.spark-submit \
+  spark-client.submit \
+    --username=ie-test \
+    --log-level "DEBUG" \
     --master $K8S_MASTER_URL \
     --deploy-mode cluster \
     --conf spark.kubernetes.driver.request.cores=100m \
     --conf spark.kubernetes.executor.request.cores=100m \
     --class org.apache.spark.examples.SparkPi \
     local:///opt/spark/examples/jars/$SPARK_EXAMPLES_JAR_NAME 100
 
-  kubectl --kubeconfig=${KUBE_CONFIG} get pods
+  # kubectl --kubeconfig=${KUBE_CONFIG} get pods
   DRIVER_JOB=$(kubectl --kubeconfig=${KUBE_CONFIG} get pods | grep driver | tail -n 1 | cut -d' ' -f1)
+
+  if [[ "${DRIVER_JOB}" == "${PREVIOUS_JOB}" ]]
+  then
+    echo "ERROR: Sample job has not run!"
+    exit 1
+  fi
+
   echo -e "Inspecting logs for driver job: ${DRIVER_JOB}"
-  kubectl --kubeconfig=${KUBE_CONFIG} logs ${DRIVER_JOB}
+  # kubectl --kubeconfig=${KUBE_CONFIG} logs ${DRIVER_JOB}
 
   EXECUTOR_JOB=$(kubectl --kubeconfig=${KUBE_CONFIG} get pods | grep exec | tail -n 1 | cut -d' ' -f1)
   echo -e "Inspecting state of executor job: ${EXECUTOR_JOB}"
-  kubectl --kubeconfig=${KUBE_CONFIG} describe pod ${EXECUTOR_JOB}
+  # kubectl --kubeconfig=${KUBE_CONFIG} describe pod ${EXECUTOR_JOB}
 
   # Check job output
-  pi=$(kubectl --kubeconfig=${KUBE_CONFIG} logs $(kubectl --kubeconfig=${KUBE_CONFIG} get pods | tail -n 1 | cut -d' ' -f1)  | grep 'Pi is roughly' | rev | cut -d' ' -f1 | rev | cut -c 1-4)
+  pi=$(kubectl --kubeconfig=${KUBE_CONFIG} logs $(kubectl --kubeconfig=${KUBE_CONFIG} get pods | tail -n 1 | cut -d' ' -f1)  | grep 'Pi is roughly' | rev | cut -d' ' -f1 | rev | cut -c 1-3)
   echo -e "Spark Pi Job Output: \n ${pi}"
```

Comment:
would be good to see some test assertions in here, so that we can have a parseable, structured report on the tests run